### ISSUE TITLE
Cannot remove `broadcast` flag from a room via edit room info dialog

### ISSUE THREAD

**Issue by jschirrmacher** [RocketChat/Rocket.Chat#12492]
### Description:
Cannot remove broadcast flag from a room via edit room info dialog

### Steps to reproduce:
Create a room with broadcast flag switched on
Go to edit room information dialog
Expected behavior:
I can remove the broadcast flag by clicking the switch.

### Actual behavior:
The switch looks normal, but doesn't change when clicked because it is disabled

### Server Setup Information:
Version of Rocket.Chat Server: 0.70.4

**Comment by ggazzo** [comment]
hey @jschirrmacher actually its not a bug, we create the broadcast as you describe, maybe change the toggle to be more "understandable" that you cant change the broadcast type... is it a problem, not be able to change the type?

**Comment by jschirrmacher** [comment]
It sure is a problem if you want to change the behaviour of a channel, because there is no other way to do it. Is there anything what makes it desirable not to be able to change the broadcast type of a channel? I don't quite understand why this is not even a bug.

**Comment by michaelthurm** [comment]
i agree, it is a problem. because usecases of channels changes by time and its not so good to loose all members of a broadcast list, when you have to open a new channel to make new messages available for all members

**Comment by avoidik** [comment]
can I disable the broadcast property of a room via rocketchat api?

**Comment by AMMAR-62** [comment]
Is anyone working on this issue??
